File: background-jobs/cron/python/job.py
import os
import time
import schedule
from datetime import datetime, timezone


def run_job() -> None:
    print(f"Python cron job ran {datetime.now(timezone.utc).isoformat()}", flush=True)


schedule.every(int(os.getenv("CRON_INTERVAL_SECONDS", "300"))).seconds.do(run_job)
run_job()

while True:
    schedule.run_pending()
    time.sleep(1)

File: background-jobs/cron/node/job.js
import cron from "node-cron";

const schedule = process.env.CRON_SCHEDULE ?? "*/5 * * * *";

cron.schedule(schedule, () => {
  console.log("Node cron job ran", new Date().toISOString());
});

console.log(`Node cron scheduled: ${schedule}`);
